app/assets/stylesheets/_layout.scss

```diff
@@ -1,7 +1,8 @@
-.content {
-  @include outer-container(68em);
+body {
+  background-color: $base-background-color;
+}
 
-  .high_voltage-pages {
-    @include outer-container(100%);
-  }
+.content {
+  margin-top: 2 * $base-spacing;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets